components/vf-hero/CHANGELOG.md

```diff
@@ -1,15 +1,5 @@
-# 0.1.0
+## 1.2.0
```

Comment:
Why the jump to 1.2.0? To make it instep with vf-card?

Reply:
yah, pretty much - as it's a sweeping change - I thought bumping it all up made sense.